tests/fcgi-responder.c

```diff
@@ -44,6 +44,10 @@ int main (void) {
 			printf("%s", getenv("PATH_INFO"));
 		} else if (0 == strcmp(p, "script_name")) {
 			printf("%s", getenv("SCRIPT_NAME"));
+		} else if (0 == strcmp(p, "var")) {
+			char *test = getenv("LIGHTYTEST");
+			if (test == NULL) test = "(no value)";
+			printf("%s", test);
```

Comment:
I added a new commit to this branch fixing these issues.
You first commented that authorizer doesn't need the Content-Type header, so I removed it. All tests run fine. Or should I add it again, as you did?